sdk/python/feast/config.py

```diff
@@ -117,69 +115,66 @@ def __init__(
         self._config = config  # type: ConfigParser
         self._path = path  # type: str
 
-    def get(self, option):
+    def _get(self, option, default, get_method):
+        fallback = {} if default is _UNSET else {"fallback": default}
+        return get_method(
+            CONFIG_FILE_SECTION,
+            option,
+            vars={**_get_feast_env_vars(), **self._options},
+            **fallback,
+        )
+
+    def get(self, option, default=_UNSET):
```

Comment:
@mike0sv How are you planning for people to use this `default` argument? Do you want them to define defaults throughout the code base, for example `.get(key, default="foo")`?

Reply:
Yes, his option can be used if there is no value even in `.constants`. Also this allows to get arbitrary options from config (it's a side effect, so I don't know if cases where this is needed exists). 
For now, if you try to get an option, which is declared as `Optional` in `.constants` (more precisely, that has `None` as default value), you'll get an error (because of the filtering in the `defaults` method)